MYTableViewIndex/Public/TableViewIndex.swift

```diff
@@ -155,6 +160,41 @@ open class TableViewIndex : UIControl {
             item.applyAttributes(style)
         }
     }
+    
+    private func selectIndex(_ index: Int) {
+        guard index != currentIndex else { return }
+        
+        currentIndex = index
+        
+        updateAccessibilityValue()
+        
+        if let delegate = self.delegate
+            , delegate.responds(to: #selector(TableViewIndexDelegate.tableViewIndex(_:didSelect:at:))) {
+            
+            let shouldProduceFeedback = delegate.tableViewIndex!(self, didSelect: items[index], at: index)
+            if shouldProduceFeedback {
+                notifyFeedbackGenerator()
+            }
+        }
+    }
+    
+    private func updateAccessibilityValue() {
+        if items.count >= currentIndex && currentIndex < items.count  {
```

Comment:
I guess it should be `currentIndex  >= 0 && currentIndex < items.count`

Reply:
Good catch, that is indeed what it should be, updated and pushed